apps/frontend/src/libs/components/quiz-question/quiz-question.tsx

```diff
@@ -1,26 +1,24 @@
 import {
-	type Control,
 	type FieldPath,
 	type FieldValues,
+	type FormFieldProperties as Properties,
 } from "~/libs/types/types.js";
 
 import { Input } from "../input/input.js";
 import styles from "./styles.module.css";
 
-type Properties<T extends FieldValues> = {
-	control: Control<T, null>;
-};
-
 const QuizQuestion = <T extends FieldValues>({
 	control,
+	name,
+	options = [],
 }: Properties<T>): JSX.Element => {
```

Comment:
Where does Properties type come from? You seem to remove it

Reply:
Properties come from imports 
We have them as common type as you suggested so I reused it.